javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  const nameArray = [];
+
+  for (const keys in students) {
+    //  console.log("value of keys:", students[keys].name);
+    nameArray.push(students[keys].name);
+  }
```

Comment:
You're actually looping over an array here, so I would probably use a `for` loop instead of a `for..in`. As we covered, `for..in` will work since Arrays technically are objects, but we really only want to use `for..in` when we are specifically looping over an object